src/lib/Libifl/int_submit.c

```diff
@@ -234,14 +234,15 @@ PBSD_rdytocmt(int c, char *jobid, int prot, char **msgid)
  * @param[in] jobid - job identifier
  * @param[in] prot - PROT_TCP or PROT_TPP
  * @param[in] msgid - message id
+ * @param[in] dest - exec_vnode if the job needs to be run
  *
  * @return      int
  * @retval      0               success
  * @retval      !0(pbs_errno)   failure
  *
  */
 int
-PBSD_commit(int c, char *jobid, int prot, char **msgid)
+PBSD_commit(int c, char *jobid, int prot, char **msgid, char *dest)
```

Comment:
I renamed the field to extend and it will accept comma-separated key value pairs.
This is for the move job to say continue run the job on specified execvnode.